conf/solr/conf/enumsConfig.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<enumsConfig>
+  <enum name="ebookAccess">
+    <value>no_ebook</value>
+    <value>unclassified</value>
+    <value>printdisabled</value>
+    <value>borrowable</value>
+    <value>public</value>
```

Comment:
Overall PR is such a great step in the right direction, very excited!

I think there may be a chance for us to mirror the terminology of Archive.org a bit more closely to reduce questions in the future, e.g.:

```
is_readable
is_lendable
is_printdisabled
is_previewable
```

We don't have to use these fields exactly, but e.g. `lendable` (the quality of whether something is e.g. in a lending program) may be both clearer and more consistent with other usages than `borrowable` (which might be used to imply the active availability of a lendable title). And have slightly less cognitive overhead since this is what Archive.org ground truth and bulk availability use.

The other one we may wish to consider is possible alternatives to `unclassified`. Maybe `restricted` may make more sense than `unclassified`? Open Library and partner services may use these values and our UI has to make _some_ call on how such catalog entries are rendered (even if this render-time decision may differ from the ground-truth value). In these cases where some item may be available and a reliable classification is not accessible through the API, I think `restricted` makes more sense as a default (which we acknowledge could sometimes have drift) than `unclassified` as I feel it better implies how we intend to (and generally do) treat them, rather than suggesting a class of items may be floating without classification.

Reply:
(Been discussing internally/on slack)